.github/workflows/verify-generated.yaml

```diff
@@ -17,13 +17,20 @@ on:
 
 jobs:
   csi-images-manifest:
-    name: verify-csi-images-manifest
+    name: verify-generated-changes
     strategy:
       matrix:
         os: [ubuntu-latest, macos-latest]
     runs-on: ${{ matrix.os }}
     steps:
       - uses: actions/checkout@v4
 
+      - uses: actions/setup-go@v5
+        with:
+          go-version-file: go.mod
+
       - name: Verify changes to CSI images manifest
         run: make verify-csi-images-manifest
+
+      - name: Verify go deps
```

Comment:
just a thought, seeing the actions log it seems we are going to download deps everytime and when you run the same in your machine it'll get copied from global cache afaik.

do you think it's worth to cache the dependencies as well?

Reply:
disregard, I just [saw](https://github.com/actions/setup-go?tab=readme-ov-file#caching-dependency-files-and-build-outputs) cache is on by default